client/src/pages/course/teams.tsx

```diff
@@ -0,0 +1,48 @@
+import { getCourseProps, noAccessResponse } from 'modules/Course/data/getCourseProps';
+import withSession from 'components/withSession';
+import { SessionProvider } from 'modules/Course/contexts';
+import { CoursePageProps } from 'services/models';
+import { GetServerSideProps } from 'next';
+import { ProfileCourseDto, TeamDistributionApi, TeamDistributionDetailedDto } from 'api';
+import { Teams } from 'modules/Teams';
+import { getTokenFromContext } from 'utils/server';
+import { getApiConfiguration } from 'utils/axios';
+
+export interface TeamsPageProps extends CoursePageProps {
+  teamDistributionDetailed: TeamDistributionDetailedDto;
+}
+
+export const getServerSideProps: GetServerSideProps<{
+  course: ProfileCourseDto;
+  teamDistributionDetailed: TeamDistributionDetailedDto;
+}> = async ctx => {
+  const courseProps = (await getCourseProps(ctx)) as any;
```

Comment:
better type properly than using `any` in this context

Reply:
will be fixed in the future